### ISSUE TITLE
org.quattor.pan:panc-maven-plugin

### ISSUE THREAD

**Issue by jrha** [quattor/release#53]
Every time I run Maven I get lots of messages like the following (one per project):
```
[WARNING] Some problems were encountered while building the effective model for org.quattor.client.libraries:perl-CAF:pom:14.8.0-rc5
[WARNING] 'build.plugins.plugin.version' for org.quattor.pan:panc-maven-plugin is missing.
[WARNING] 
[WARNING] It is highly recommended to fix these problems because they threaten the stability of your build.
[WARNING] 
[WARNING] For this reason, future Maven versions might no longer support building such malformed projects.
```
Do we actually need to include the `panc-maven-plugin` or is it a copy-paste artefact?

**Comment by jouvin** [comment]
In fact this was fixed by the cleanups done in build tools 1.42 (and related cleanups in pom files used by various components)...

**Comment by jrha** [comment]
Great news!
